src/firmware_new/proto/control.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto2";
+
+// A test robot msg which serves as an example to modify and use to
+// facilitate firmware testing.
+
+message control_msg {
+
+    // the 4 wheel speeds are optional, if no speed is provided,
+    // then the previous speed is maintained
```

Comment:
nit; how the robot responds to a message should not be part of the message. A message contains information, shouldn't dictate what to do with that information.

Reply:
removed